staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -150,6 +150,7 @@ type TLSConfig struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // TracingConfiguration provides versioned configuration for tracing clients.
+// This type is deprecated in favor of component-base/config TracingConfiguration
```

Comment:
I'm not sure what is meant by this ... what action is expected from apiserver consumers of this config struct in response to this deprecation comment?

The component-base TracingConfiguration type doesn't have TypeMeta in it, so it's not a standalone serializable type, so consumers couldn't actually switch to using the component-base version in a file

Reply:
Same question on staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

Reply:
got it, removed the comments